### PR TITLE
Fixed alignment issues on small mobile device AB#11811

### PR DESCRIPTION
**Pull request by pagustinqt20**
# Fixed alignment issues on small mobile device [AB#11811](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/11811)

## Description

Fixed alignment issues on small mobile devices. 

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [x] Not Required

### UI Changes

**YES** 

**Note: force enabled the buttons in local to check the alignment.**

image

image

image

image

image

### Browsers Tested

-   [x] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

**Review by MikeLyttle — COMMENTED** [review]
Does this solution work at all widths?  I'm thinking that the button might wrap to the next line not exactly at the `md` breakpoint, leaving odd spacing in that instance.

That could be addressed by wrapping the line in a `<b-row>`, with one column for the input and another for the button.  The button's column could be made full-width below the `md` breakpoint by using `<b-col cols="12" md="auto">`.

Also, a nit: I think `mt-2` might look nicer than `mt-1` for the vertical spacing.

**Comment by pagustinqt20** [comment]
> Does this solution work at all widths? I'm thinking that the button might wrap to the next line not exactly at the `md` breakpoint, leaving odd spacing in that instance.
> 
> That could be addressed by wrapping the line in a `<b-row>`, with one column for the input and another for the button. The button's column could be made full-width below the `md` breakpoint by using `<b-col cols="12" md="auto">`.
> 
> Also, a nit: I think `mt-2` might look nicer than `mt-1` for the vertical spacing.

**Comment by pagustinqt20** [comment]
@MikeLyttle I checked all the mobile sizes in the dropdown and it shows okay. Let me try your suggestion

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 1]
```diff
@@ -538,39 +538,47 @@ export default class ProfileView extends Vue {
...
+                                                        isValid($v.email) ||
+                                                        !isEmailEditable
+                                                            ? null
+                                                            : false
+                                                    "
+                                                />
+                                            </b-col>
+                                            <b-col
+                                                cols="12"
+                                                md="auto pl-md-0 pl-3"
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:564]:
```
md="auto pl-md-0 pl-3"
```

Should be:

```
md="auto"
class="pl-md-0 pl-3"
```

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 2]
```diff
@@ -687,39 +695,45 @@ export default class ProfileView extends Vue {
...
+                                                        isValid($v.smsNumber) ||
+                                                        !isSMSEditable
+                                                            ? null
+                                                            : false
+                                                    "
+                                                />
+                                            </b-col>
+                                            <b-col
+                                                cols="12"
+                                                md="auto pl-md-0 pl-3"
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:724]:
```
md="auto pl-md-0 pl-3"
```

Should be:

```
md="auto"
class="pl-md-0 pl-3"
```

**Review by MikeLyttle — CHANGES_REQUESTED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**